discovery/src/main/java/com/ibm/watson/developer_cloud/discovery/v1/model/LanguageEnrichmentOptions.java

```diff
@@ -0,0 +1,254 @@
+/*
+ * Copyright 2017 IBM Corp. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package com.ibm.watson.developer_cloud.discovery.v1.model;
+
+import com.google.gson.annotations.SerializedName;
+import com.ibm.watson.developer_cloud.service.model.GenericModel;
+
+/**
+ * options specific to the `alchemy_language` enrichment.
+ */
+public class LanguageEnrichmentOptions extends GenericModel {
+
+  /**
+   * If provided, then do not attempt to detect the language of the input document. Instead, assume the language is
+   * the one specified in this field.  You can set this property to work around `unsupported-text-language` errors.
+   * Supported languages include English, German, French, Italian, Portuguese, Russian, Spanish and Swedish.
+   * Supported language codes are the ISO-639-1, ISO-639-2, ISO-639-3, and the plain english name of the language (e
+   * .g. "russian").
+   */
+  public enum Language {
```

Comment:
There is a `LanguageEnrichmentOptions` model in the swagger.  But it seems that it's only use is in a "$allOf" in `EnrichmentOptions`.  So we should probably excluded it from the generated SDK.